epifor/gleam/gleamdef.py

```diff
@@ -139,28 +135,20 @@ def set_beta(self, val):
             './gv:definition/gv:compartmentalModel/gv:variables/gv:variable[@name="beta"]'
         ).set("value", f"{val:.2f}")
 
-    def get_air_traffic(self):
-        return (
-            float(self.f1("./gv:definition/gv:parameters").get("occupancyRate")) / 100.0
-        )
-
-    def set_air_traffic(self, val):
-        assert val < 1.1
-        v = int(max(min(val * 100, 100), 0))
-        self.f1("./gv:definition/gv:parameters").set("occupancyRate", str(v))
-
     def get_traffic_occupancy(self):
+        "Note: this an integer in percent"
```

Comment:
Use triple quotes for docstrings (this is technically just an unassigned "string"). It's at several places.

```suggestion
        """Note: this an integer in percent"""
```

Reply:
Nah. But thanks.
Technically it is the same.